Shared/ActiveDirectoryFunctions/Get-ExchangeWebSitesFromAd.ps1

```diff
@@ -0,0 +1,44 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+. $PSScriptRoot\Get-ExchangeProtocolContainer.ps1
+
+function Get-ExchangeWebSitesFromAd {
+    [CmdletBinding()]
+    [OutputType([System.Object])]
+    param (
+        [string]$ComputerName = $env:COMPUTERNAME
+    )
+
+    begin {
+        function GetExchangeWebSiteFromCn {
+            param (
+                [string]$Site
+            )
+
+            if ($null -ne $Site) {
+                $index = $Site.IndexOf("(") + 1
+                if ($index -ne -1) {
```

Comment:
This will never be -1, because the line above is adding 1. Needs to change to be 0 instead.

Reply:
Indeed 👍 . Fixed